app/Libraries/BeatmapsetDiscussionReview.php

```diff
@@ -97,6 +101,22 @@ public static function create(Beatmapset $beatmapset, array $document, User $use
             BeatmapDiscussion::whereIn('id', $childIds)
                 ->update(['parent_id' => $review->getKey()]);
 
+            // handle disqualifications and the resetting of nominations
+            if ($problemPost) {
+                $resetNominations = $beatmapset->isPending() &&
+                    $beatmapset->hasNominations() &&
+                    priv_check_user($user, 'BeatmapsetResetNominations', $beatmapset)->can();
+
+                if ($resetNominations) {
+                    BeatmapsetEvent::log(BeatmapsetEvent::NOMINATION_RESET, $user, $problemPost)->saveOrExplode();
```

Comment:
looks like this should [broadcast reset nomination event](https://github.com/ppy/osu-web/blob/04df7c7a15f96357f3631a0157655a7661a54f70/app/Http/Controllers/BeatmapDiscussionPostsController.php#L170) as well?

There's also missing broadcast of [global notification on first new problem on qualified beatmapset](https://github.com/ppy/osu-web/blob/04df7c7a15f96357f3631a0157655a7661a54f70/app/Http/Controllers/BeatmapDiscussionPostsController.php#L185).